models/public/device_support.md

```diff
@@ -91,6 +91,7 @@
 | mobilenet-v2-pytorch | YES | YES | YES |
 | mobilenet-v3-large-1.0-224-tf | YES | YES | YES |
 | mobilenet-v3-small-1.0-224-tf | YES | YES | YES |
+| mobilenet-yolo-v4-syg | YES |    |    |
```

Comment:
I was able to successfully test the model on iGPU, so we can put YES for GPU column (did not try MYRIAD yet)